pkg/filter/filter_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"testing"
 
 	. "gopkg.in/check.v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Not needed?

Reply:
VSCode likely added it automatically. I don't think it is needed. OTOH, 72 of the 82 imports of `"k8s.io/api/core/v1"` in kanister code use it. So I'd do it for consistency.